src/core/unit.c

```diff
@@ -1311,27 +1311,26 @@ int unit_add_exec_dependencies(Unit *u, ExecContext *c) {
          * is run first. */
 
         if (c->log_namespace) {
-                FOREACH_STRING(s, "systemd-journald", "systemd-journald-varlink") {
-                        _cleanup_free_ char *socket_unit = NULL;
+                static const struct {
+                        const char *template;
+                        UnitType type;
+                } deps[] = {
+                        { .template = "systemd-journald",         .type = UNIT_SOCKET,  },
```

Comment:
Hmm, it's not necessary to explicitly write field name here?

Reply:
Maybe? But that does not hurt anything? If you really do not like it, then I am OK to drop them. But otherwise I'd like to keep them.

Reply:
I think it's a bit too verbose (but doesn't really matter)

Reply:
OK. Dropped now.